tools/pkg/engine_build_configs/lib/src/build_config.dart

```diff
@@ -601,6 +617,21 @@ final class GlobalTest extends BuildConfigBase {
   /// A list of dictionaries representing scripts and parameters to run them
   final List<TestTask> tasks;
 
+  /// Returns true if platform is capable of executing this build and false
+  /// otherwise.
+  bool canRunOn(Platform platform) {
+    String? os;
+    for (final String dimension in droneDimensions) {
+      os ??= switch (dimension.split('=')) {
+        ['os', 'Linux'] => Platform.linux,
+        ['os', 'Windows'] => Platform.windows,
```

Comment:
It looks like we'll need to support `Windows-10` and `Windows-11`. Example: https://github.com/flutter/engine/blob/41daa5b913b928129d7b370012b8ccb57054e72d/ci/builders/windows_host_engine.json#L22

Reply:
Oh, good catch. Done.